index.html

```diff
@@ -230,65 +230,114 @@ <h3 class="works-grid-one-project-header">Profesional Art Printing Data</h3>
+          <li class="skill-lang-li">Ruby</li>
+          <li class="skill-lang-li">Html</li>
+          <li class="skill-lang-li">Css</li>
+          </ul>
 
-        <ul class="skills-lang">
-        <li class="skill-lang-header">
-            <span class="square"><svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
-                <rect y="0.455811" width="18" height="18" fill="#FF6B00"></rect>
-            </svg> </span>Frameworks
-        </li>
-        <li class="skill-lang-li">Bootstrap</li>
-        <li class="skill-lang-li">Ruby on Rails</li>
-        <li class="skill-lang-li">RSPec</li>
-        <li class="skill-lang-li">CapyBara</li>
-        <li class="skill-lang-li">Selenium</li>
-        </ul>
+          <ul class="skills-lang">
+          <li class="skill-lang-header">
+              <span class="square"><svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <rect y="0.455811" width="18" height="18" fill="#FF6B00"></rect>
+              </svg> </span>Frameworks
+          </li>
+          <li class="skill-lang-li">Bootstrap</li>
+          <li class="skill-lang-li">Ruby on Rails</li>
+          <li class="skill-lang-li">RSPec</li>
+          <li class="skill-lang-li">CapyBara</li>
+          <li class="skill-lang-li">Selenium</li>
+          </ul>
 
-        <ul class="skills-lang">
-        <li class="skill-lang-header">
-            <span class="circle"><svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
-                <rect y="0.455811" width="18" height="18" rx="9" fill="#FF6B00"></rect>
-            </svg> </span>Skills
-        </li>
-        <li class="skill-lang-li">Codekit</li>
-        <li class="skill-lang-li">Github</li>
-        <li class="skill-lang-li">Codepen</li>
-        <li class="skill-lang-li">Gitlab</li>
-        <li class="skill-lang-li">Terminal</li>
-        </ul>
-    </div>
+          <ul class="skills-lang">
+          <li class="skill-lang-header">
+              <span class="circle"><svg width="18" height="19" viewBox="0 0 18 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <rect y="0.455811" width="18" height="18" rx="9" fill="#FF6B00"></rect>
+              </svg> </span>Skills
+          </li>
+          <li class="skill-lang-li">Codekit</li>
+          <li class="skill-lang-li">Github</li>
+          <li class="skill-lang-li">Codepen</li>
+          <li class="skill-lang-li">Gitlab</li>
+          <li class="skill-lang-li">Terminal</li>
+          </ul>
+      </div>
+    </section>
+    <!--end-->
+    <!--form section-->
+    <section class="contact" id="contact">
+      <div>
+        <h2 class="contact-description">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+      </div>
+
+      <form action="#" method="POST" id="form">
```

Comment:
- [ ] Kindly change the action to be something similar to 
```
https://formspree.io/f/{form_id}
```
You can use the [service](https://formspree.io/html/) provided in the project requirement to generate your link.